nowcasting_dataset/consts.py

```diff
@@ -55,4 +55,7 @@
 NWP_Y_COORDS = "nwp_y_coords"
 X_METERS_CENTER = "x_meters_center"
 Y_METERS_CENTER = "y_meters_center"
+TOPOGRAPHIC_DATA = "topo_data"
+TOPOGRAPHIC_X_COORDS = "topo_x_coords"
+TOPOGRAPHIC_Y_COORDS = "topo_y_coords"
 T0_DT = "t0_dt"
```

Comment:
nice

I think these might need to be added in a few other places (that ive only discoverd by playing with stuff)

datatypes - https://github.com/openclimatefix/nowcasting_dataset/blob/main/nowcasting_dataset/dataset/batch.py#L42

batch to dataset - https://github.com/openclimatefix/nowcasting_dataset/blob/main/nowcasting_dataset/dataset/batch.py#L70

required_keys - https://github.com/openclimatefix/nowcasting_dataset/blob/main/nowcasting_dataset/dataset/datasets.py#L113


validate dataset
https://github.com/openclimatefix/nowcasting_dataset/blob/main/nowcasting_dataset/dataset/example.py#L109

Reply:
Done!